### ISSUE TITLE
Use Github Actions dependency cache to make CI tests run faster

### ISSUE THREAD

**Issue by augusto-herrmann** [economiagovbr/Ro-dou#45]
Github Actions now supports using [dependency caching](https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows) to reuse a container after all dependencies have been installed.

When a workflow uses this, it enables test to run much faster, as no time is wasted building and installing dependencies every time the CI workflow is run.

**Comment by nitaibezerra** [comment]
I think Ro-dou should have it's own Dockerfile and be maintained independent from airflow2-docker image. It is already necessary in my case. I'm developing an improvement using an extra lib which, in the current situation, has to be added to the airflow2-docker image.

I'll create this new Issue.

So maybe this one should be reopened.

**Comment by augusto-herrmann** [comment]
Agreed. After #48 is solved, we can reevaluate if a cache would be useful and, if so, reopen this.
